easybuild/easyconfigs/v/velocyto.py/velocyto.py-0.17.17-intel-2020a-Python-3.8.2.eb

```diff
@@ -0,0 +1,55 @@
+easyblock = 'PythonBundle'
+
+name = 'velocyto.py'
+version = '0.17.17'
+versionsuffix = "-Python-%(pyver)s"
+
+homepage = 'https://velocyto.org/velocyto.py/'
+description = "Velocyto is a library for the analysis of RNA velocity."
+
+toolchain = {'name': 'intel', 'version': '2020a'}
+
+dependencies = [
+    ('Python', '3.8.2'),
+    ('SciPy-bundle', '2020.03', versionsuffix),
+    ('matplotlib', '3.2.1', versionsuffix),
+    ('numba', '0.50.0', versionsuffix),
+    ('scikit-learn', '0.23.1', versionsuffix),
+    ('h5py', '2.10.0', versionsuffix),
+    ('Pysam', '0.16.0.1'),
+
+]
+
+use_pip = True
+
+exts_default_options = {'source_urls': [PYPI_SOURCE]}
+
+exts_list = [
+    ('numpy-groupies', '0.9.13', {
+        'modulename': 'numpy_groupies',
+        'source_tmpl': 'numpy_groupies-%(version)s.tar.gz',
```

Comment:
You can remove this too if you use `numpy_groupies` as name instead two lines higher